src/utilities.jl

```diff
@@ -1,12 +1,17 @@
 # VSCode specific
 # ---------------
 
-nodocument_error(uri, data=nothing) =
-    return JSONRPC.JSONRPCError(-32099, "document $(uri) requested but not present in the JLS", data)
+function nodocument_error(uri, data=nothing)
+    return JSONRPC.JSONRPCError(
+        1000,
+        "document $(uri) requested but not present in the JLS",
+        data
+    )
+end
 
 function mismatched_version_error(uri, doc, params, msg, data=nothing)
     return JSONRPC.JSONRPCError(
-        -32099,
+        1001,
```

Comment:
Thoughts on keeping in line with error codes in the -32k (ish) range? E.g. `-32101` etc?

Reply:
I’m reading the spec as saying those are all reserved for future use and therefore shouldn’t be used by us?

Reply:
Ah, right, skipped over that paragraph. -33xxx then?

Reply:
I have zero preferences, so also happy to use that range. I picked the ones in this PR because they seemed far away from anything anyone else might use. Are there conversations for negative numbers around, in general?

Reply:
Alright, now with new error codes :)